From 95fcb98d0189c3f9a051b56416503b42ac29ff90 Mon Sep 17 00:00:00 2001
From: Christoph J. Thompson <cjsthompson@gmail.com>
Date: Tue, 11 Jan 2011 16:17:48 +0100
Subject: [PATCH 1/2] Port to libxfce4ui.

	modified:   configure.ac
	modified:   panel-plugin/Makefile.am
	modified:   panel-plugin/netload.c
---
 configure.ac             |    2 +-
 panel-plugin/Makefile.am |    4 ++--
 panel-plugin/netload.c   |    5 +++--
 3 files changed, 6 insertions(+), 5 deletions(-)

diff --git a/configure.ac b/configure.ac
index 0348bfa..3baaa14 100644
--- a/configure.ac
+++ b/configure.ac
@@ -56,7 +56,7 @@ dnl configure the panel plugin
 XDT_CHECK_PACKAGE([LIBXFCE4PANEL], [libxfce4panel-1.0], [4.3.20])
 
 dnl configure the libxfcegui4
-XDT_CHECK_PACKAGE([LIBXFCEGUI4], [libxfcegui4-1.0], [4.4.0])
+XDT_CHECK_PACKAGE([LIBXFCE4UI], [libxfce4ui-1], [4.7.0])
 
 dnl Check for i18n support
 XDT_I18N([@LINGUAS@])
diff --git a/panel-plugin/Makefile.am b/panel-plugin/Makefile.am
index 0635d67..2163277 100644
--- a/panel-plugin/Makefile.am
+++ b/panel-plugin/Makefile.am
@@ -17,12 +17,12 @@ xfce4_netload_plugin_SOURCES =						\
 xfce4_netload_plugin_CFLAGS =						\
 	-DPACKAGE_LOCALE_DIR=\"$(localedir)\"				\
 	@LIBXFCE4PANEL_CFLAGS@				\
-	@LIBXFCEGUI4_CFLAGS@
+	@LIBXFCE4UI_CFLAGS@
 
 xfce4_netload_plugin_LDADD =						\
 	@SOLLIBS@                                   			\
 	@LIBXFCE4PANEL_LIBS@                                   			\
-	@LIBXFCEGUI4_LIBS@
+	@LIBXFCE4UI_LIBS@
 
 # .desktop file
 #
diff --git a/panel-plugin/netload.c b/panel-plugin/netload.c
index b4868c4..43035f2 100644
--- a/panel-plugin/netload.c
+++ b/panel-plugin/netload.c
@@ -30,7 +30,7 @@
 #include <gtk/gtk.h>
 
 #include <libxfce4util/libxfce4util.h>
-#include <libxfcegui4/libxfcegui4.h>
+#include <libxfce4ui/libxfce4ui.h>
 #include <libxfce4panel/xfce-panel-plugin.h>
 
 
@@ -460,7 +460,8 @@ static void setup_monitor(t_global_monitor *global, gboolean supress_warnings)
     if (!init_netload( &(global->monitor->data), global->monitor->options.network_device)
             && !supress_warnings)
     {
-        xfce_err (_("%s: Error in initalizing:\n%s"),
+        xfce_dialog_show_error (NULL, NULL,
+	    _("%s: Error in initalizing:\n%s"),
             _(APP_NAME),
             _(errormessages[
                 global->monitor->data.errorcode == 0 
-- 
1.7.1

